src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -76,8 +76,8 @@ void AppHost::Initialize()
     _app.TitleChanged({ this, &AppHost::AppTitleChanged });
     _app.LastTabClosed({ this, &AppHost::LastTabClosed });
 
-    AppTitleChanged(_app.GetTitle());
-
+    _window->UpdateTitle(_app.GetTitle().c_str());

```

Comment:
Do we need `c_str` because UpdateTitle takes a wstring?

Reply:
removed, no need.